snapcraft/plugins/go.py

```diff
@@ -39,6 +39,10 @@
       This entry tells the checked out `source` to live within a certain path
       within `GOPATH`.
       This is not needed and does not affect `go-packages`.
+
+    - go-buildtags:
+      (string)
```

Comment:
> go-tags feels confusing to me, but if both of you prefer it fine by me ;-)

Not necessarily, but if we ever intend on supporting other cli options for `go install`, naming some of them corresponding to the param in question and others different might make for a confusing experience.